carbon-design-system/carbon
# fix(OverflowMenu): changed filename for story

Closes #9804 

Cribbed a function from the old OverflowMenu to `preventDefault` on all the arrow keys when the state is set to open.

#### Testing

1) Open the next storybook
2) Navigate to the [unstable OverflowMenu](https://deploy-preview-10041--carbon-react-next.netlify.app/?path=/story/experimental-unstable-menu-overflowmenu--overflow-menu)
3) Inspect the element with your devtools and toss in `height: 42rem` or some other ridiculous amount of rem.
4) Open the overflow menu with either your mouse or keyboard, then use your keyboard to navigate the menu items.
5) It should be as expected instead of the menu items detaching from icon when you navigate

#### Questions

I feel like this code should be working, but for the life of me, it isn't. Would love an extra set of eyes on this :)
